app/views/content/steps-to-become-a-teacher/_step_8_start_your_career.erb

```diff
@@ -2,8 +2,8 @@
 
 <p>Now that you’ve started your first teaching job, you’ll be an early career teacher (ECT) – this used to be called newly qualified teacher (NQT).</p>
 
-<p>For your first 2 years as an early career teacher, you’ll receive a package of support to help you find your feet.</p>
+<p>For your first 2 years as an early career teacher, you’ll receive a package of support to help you find your feet. You'll get this automatically if you work in a maintained school but lots of other schools provide this for ECTs too.</p>
 
-<p>You’ll get a mentor to coach you, boost your confidence and help you develop key skills including behaviour management.</p>
+<p>You’ll get a mentor to support you, boost your confidence and help you develop key skills, including behaviour management and ways to adapt your teaching, to build on your initial teacher training.</p>
```

Comment:
```suggestion
<p>You’ll get a mentor to support you and boost your confidence. They'll also help you develop key skills, including behaviour management and ways to adapt your teaching, building on your initial teacher training.</p>
```

@MaxineCl Wonder if we need to break this up now, as it has got quite long? But do disagree/reword as needed!

Reply:
@gemmadallmandfe I've split it into 2 sentences